wknml/__init__.py

```diff
@@ -1,26 +1,40 @@
 import xml.etree.ElementTree as ET
 from loxun import XmlWriter
 from typing import NamedTuple, List, Tuple, Optional
+import collections
 
 Vector3 = Tuple[float, float, float]
 Vector4 = Tuple[float, float, float, float]
 IntVector6 = Tuple[int, int, int, int, int, int]
 
-NMLParameters = NamedTuple(
+# From https://stackoverflow.com/a/18348004
+# Use the defaults parameter when switching to Python 3.6
```

Comment:
Unfortunately, we need to wait for 3.7 and not 3.6.

Reply:
I think the typing.NamedTuple module supports default values for Python 3.6.1, see https://docs.python.org/3/library/typing.html#typing.NamedTuple :)

Reply:
cool stuff